src/runtime/graph_executor/graph_executor.cc

```diff
@@ -384,9 +446,15 @@ void GraphExecutor::SetupOpExecs() {
 
     for (size_t i = 0; i < inode.inputs.size(); i++) {
       uint32_t eid = this->entry_id(inode.inputs[i]);
-      // check if op input is model input
-      if (input_node_eids.count(eid) > 0) {
-        input_dltensors_[eid].push_back(static_cast<DLTensor*>(op_args->arg_values[i].v_handle));
```

Comment:
Why does this check be moved?

Reply:
*          A    B
   *           \  /
   *      elemwise_add(out0)
   *              \
   *       C      copy
   *        \      /
   *      elemwise_sub(out1)
set_output_zero_copy(out0), we also should change the node("copy") input memory address.
I use input_dltensors_ to change node("copy") input mem address.
This check only support model input node mem addr insert to the input_dltensors_, but the node("Copy") is not model input node

Reply:
here seems like still need a condition check that if the node in inputs is model input or output, without such check all node in inputs would get push into input_dltensors_ and logically incorrect.

Reply:
Inorder to modify the node  input mem addr which connect to the output node, all nodes input will push into the input_dltensors_， so I removed the check.
Before deleting, the size of input_dltensors_ is the same as nodes_. The non-model input node input_dltensors[X] is empty. After the modification, the nodes that do not affect the model input, the non-input nodes will be push into the input_dltensors_ .

The essence is that input_dltensors_  records the input memory of all nodes, not just the memory of the model input. Does not affect the function of set_input.
I think this will not cause bugs.What do you think

Reply:
if the purpose is to modify the input node "connect to the output node" then here code should do same thing as logic asked instead of push "all node" into input_dltensors that conflict with the logic.

this logic seems like not cause a  serious bug, but first it will cause confuse issue like "why all input get push into input_dltensors_ but only partial get used that will make maintain become difficult, secondly this unnecessary
logic that tracking all node input increased the performance cost for all existing use case of GraphExecutor especially for big network case,  I think here should fix such logic issue and avoid these side effect.

Reply:
modify nid->eid to record output
add var to record the dltensor both model output op input
@huajsj review again